Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -4,7 +4,10 @@ import Prelude
 import ReactiveSwift
 import UIKit
 
+public let maxBadgeValue = 99
```

Comment:
Since we're only using this constant from the function bellow does it still make sense to have it here (as oppose to inside the function)?

If you think it's better to have it as a global constant how about we extract this to `Constants.swift` or something and maybe make it an enum?